rss_parser/_parser.py

```diff
@@ -64,7 +70,15 @@ def parse(self) -> RSSFeed:
                     for image in description_soup.findAll("img")
                 ],
             }
+
             try:
+                # Add user-defined entries
+                item_dict.update({"other": {}})
+                for entrie in entries:
+                    value = getattr(getattr(item, entrie, ""), "text", "")
+                    value = re.sub(f"</?{entrie}>", "", value)
```

Comment:
Oh github why did you delete my text? 
> 3. Removes 
> `<attribute>`
>  and 
> `</attribute>`
> from the text because they are not stripped automatically like the other conventional attributes of the rss files.

Reply:
> Oh github why did you delete my text?
> 
> > 3. Removes
> >    `<attribute>`
> >    and
> >    `</attribute>`
> >    from the text because they are not stripped automatically like the other conventional attributes of the rss files.

Nice, thanks for the explanation